src/message.js

```diff
@@ -181,23 +182,35 @@ Message.prototype.decryptSessionKeys = async function(privateKeys, passwords) {
       throw new Error('No public key encrypted session key packet found.');
     }
     await Promise.all(pkESKeyPacketlist.map(async function(keyPacket) {
-      const privateKeyPackets = new packet.List();
-      privateKeys.forEach(privateKey => {
-        privateKeyPackets.concat(privateKey.getKeys(keyPacket.publicKeyId).map(key => key.keyPacket));
-      });
-      await Promise.all(privateKeyPackets.map(async function(privateKeyPacket) {
-        if (!privateKeyPacket) {
-         return;
-        }
-        if (!privateKeyPacket.isDecrypted()) {
-          throw new Error('Private key is not decrypted.');
-        }
-        try {
-          await keyPacket.decrypt(privateKeyPacket);
-          keyPackets.push(keyPacket);
-        } catch (err) {
-          util.print_debug_error(err);
+      await Promise.all(privateKeys.map(async function(privateKey) {
+        const primaryUser = await privateKey.getPrimaryUser(/* symEncryptedPacket.created, userId */);
+        let algos = [
+          enums.symmetric.aes256, // OpenPGP.js default
```

Comment:
Should we replace this with the current config value? Or perhaps add that one as well?

Reply:
Yeah, maybe. The issue is that if you change the config value, other people using the normal default will throw -- so maybe it's better to find that out quickly.
Similarly, GPG will warn even with our normal default if we send an AES256 message to someone without AES256 in the preferred keys - so maybe this is just an argument to switch to 3DES as the fallback, as the spec mandates, make that non-configurable, and then add the config value to the start of the preferred algos when generating keys (and use it for password encryption).

Reply:
Yes, let's do that